src/app/api/api_v1/endpoints/cameras.py

```diff
@@ -55,7 +63,9 @@ async def heartbeat(
     token_payload: TokenPayload = Security(get_jwt, scopes=[Role.CAMERA]),
 ) -> Camera:
     # telemetry_client.capture(f"camera|{token_payload.sub}", event="cameras-heartbeat", properties={"camera_id": camera_id})
-    await cameras.get(token_payload.sub, strict=True)
+    camera = await cameras.get(token_payload.sub, strict=True)
+    if camera is not None and token_payload.organization_id != camera.organization_id:
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Access forbidden.")
```

Comment:
The camera is updating itself by design, so actually we can even remove the previous line. With the new version, I think we'll have a problem: if the camera is None, we're gonna get a problem when updating the camera next.

I suggest that we remove the former & newer lines. If the token is forged we have bigger problems. If it's not, it means it comes from a camera, and that camera would be updating itself. (even if somehow in the future we allow admins to update the org ID of the camera between the moment the token is created and the heartbeat, I would say we still want the camera to be able to update itself)

I don't think we should run any org verification for "self" methods

Reply:
hum I don't understand the issue, why a camera would be None ?

Reply:
Oh I got confused by the
´´´python
if camera is not None
´´´

The 'strict=True' throws an error in case the camera isn't found (none), so would you mind removing this conditional check?